### PR TITLE
Mm 3218

### PR DESCRIPTION
**Pull request by niekvangalen**

### CODE REVIEW

**Comment by niekvangalen** [comment]
@ArdonToonstra wil je hier naar kijken, en naar mijn comment in https://bits.nictiz.nl/browse/MM-3218?

File: Profiles - ZIB 2017/conceptmap-EmailSoortCodeLijst-TelecomTypeCodelijst-to-ContactPointSystem.xml [hunk 1]
```diff
@@ -3,9 +3,9 @@
     <extension url="http://hl7.org/fhir/StructureDefinition/concept-bidirectional">
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/conceptmap-EmailSoortCodeLijst-TelecomTypeCodelijst-to-ContactPointSystem.xml:3]: Forgot to change the `id`?

**Review by ArdonToonstra — COMMENTED** [review]
Looks ok!  I made one small remark inline.

**Comment by ArdonToonstra** [comment]
@niekvangalen, indeed on `.use` are even 3 ConceptMap reference extensions!  I cannot answer in MM-3218, maybe because it the status is in 'Realisatie'?

I am not completely sure if I understand why it will not be possible to combine those ValueSets. It relates to this part:

> If I merge those ConceptMaps it gets confusing, because the same codes from two different ValueSets will have (and need) a different comment value matching with the zib.

The comment value is not that important in the ConceptMap, is it? The mapping itself and the `[equivalence](https://www.hl7.org/fhir/conceptmap-definitions.html#ConceptMap.group.element.target.equivalence)` seem most relevant to me. Do you have an example/snippet of what the confusion should look like?

**Comment by niekvangalen** [comment]
@ArdonToonstra 

Snippet from the NummerSoortCodelijst-to-ContactPointUse ConceptMap:
```<element>
            <code value="HP"/>
            <target>
                <code value="home"/>
                <equivalence value="equal"/>
                <comment value="HP: Primary Home/Telefoonnummer thuis."/>
            </target>
        </element>
```
Snippet from the EmailSoortCodelijst-to-ContactPointUse ConceptMap:
```<element>
            <code value="HP"/>
            <target>
                <code value="home"/>
                <equivalence value="equal"/>
                <comment value="HP: Primary Home/Privé e-mailadres. Use in conjunction with ContactPoint.use = email."/>
            </target>
        </element>
```
The codes are the same but the comment value differs, aligning with the zib.

**Comment by niekvangalen** [comment]
@pieter-edelman-nictiz what's your opinion on the above?

**Comment by ArdonToonstra** [comment]
@niekvangalen, those two comments could be merged? Maybe something like:

``` xml
<comment value="HP: Primary Home for Telefoonnummer thuis/Privé e-mailadress. If used for Privé e-mailadres use in conjunction with ContactPoint.use = email."/>
```

**Comment by niekvangalen** [comment]
Closed, see comment in https://bits.nictiz.nl/browse/MM-3218

**Pull request closed without merging**